### ISSUE TITLE
WARNING: java.lang.ClassNotFoundException: org.geotools.data.wfs.v1_0_0.xml.WFSSchema

### ISSUE THREAD

**Issue by mprins** [flamingo-geocms/flamingo#884]
I just noticed the following in a build log:
```
Aug 01, 2017 2:05:14 PM org.geotools.xml.SchemaFactory loadSchemas
WARNING: java.lang.ClassNotFoundException: org.geotools.data.wfs.v1_0_0.xml.WFSSchema
```
This also popped up in #512

full log snippet
```
14:05:10 Flamingo-viewer-admin-test:  INFO (nl.b3p.viewer.util.LoggingTestUtil#startTest:45) - ==== Start test methode: addWMSGeoservice[naam: geoserver-wmsurl]
14:05:10 Flamingo-viewer-admin-test:  INFO (org.hsqldb.persist.Logger#logInfoEvent:?) - Checkpoint start
14:05:10 Flamingo-viewer-admin-test:  INFO (org.hsqldb.persist.Logger#logInfoEvent:?) - checkpointClose start
14:05:10 Flamingo-viewer-admin-test:  INFO (org.hsqldb.persist.Logger#logInfoEvent:?) - checkpointClose synched
14:05:10 Flamingo-viewer-admin-test:  INFO (org.hsqldb.persist.Logger#logInfoEvent:?) - checkpointClose script done
14:05:10 Flamingo-viewer-admin-test:  INFO (org.hsqldb.persist.Logger#logInfoEvent:?) - checkpointClose end
14:05:10 Flamingo-viewer-admin-test:  INFO (org.hsqldb.persist.Logger#logInfoEvent:?) - Checkpoint end - txts: 1
14:05:14 Flamingo-viewer-admin-test: DEBUG (nl.b3p.viewer.admin.stripes.WFSTypeNamingTest#addWMSGeoservice:148) - Starting WMS test with: nl.b3p.viewer.admin.stripes.WFSTypeNamingTest, serviceUrl: http://ibis.b3p.nl/geoserver/wms?SERVICE=WMS&, serviceName: geoserver-wmsurl, typeCount: 3, groupLayerCount: 1
Aug 01, 2017 2:05:14 PM org.geotools.xml.SchemaFactory loadSchemas
WARNING: java.lang.ClassNotFoundException: org.geotools.data.wfs.v1_0_0.xml.WFSSchema
14:05:15 Flamingo-viewer-admin-test: DEBUG (nl.b3p.viewer.admin.stripes.WFSTypeNamingTest#addWMSGeoservice:172) - Inspecting layer, name: ibis:bedrijvenkavels, featuretype: bedrijvenkavels
14:05:15 Flamingo-viewer-admin-test: DEBUG (nl.b3p.viewer.admin.stripes.WFSTypeNamingTest#addWMSGeoservice:172) - Inspecting layer, name: ibis:bedrijventerrein, featuretype: bedrijventerrein
14:05:15 Flamingo-viewer-admin-test: DEBUG (nl.b3p.viewer.admin.stripes.WFSTypeNamingTest#addWMSGeoservice:172) - Inspecting layer, name: ibis:v_component_ibis_report, featuretype: v_component_ibis_report
14:05:15 Flamingo-viewer-admin-test:  INFO (nl.b3p.viewer.util.LoggingTestUtil#endTest:53) - ==== Einde test methode: addWMSGeoservice[naam: geoserver-wmsurl]

```

**Comment by mprins** [comment]
see: https://osgeo-org.atlassian.net/browse/GEOT-5795

**Comment by mprins** [comment]
This warning message will be resolved once we update geotools to 17.3 or later as the patches have been applied upstream
